src/helpers/getCoursesDuration.js

```diff
@@ -1,27 +1,24 @@
-const getCoursesDuration = (duration) => {
-  const hours = Math.floor(duration / 60);
-  const minutes = duration % 60;
-  let durationResult = '00:00 hour';
+function getCoursesDuration(duration) {
```

Comment:
Why did you make there changes?
Same case for variables names

Reply:
Because previous solution was not working properly. I'll change it to AF again, I forgot.